server/src/index.js

```diff
@@ -2,20 +2,18 @@ const express = require('express')
 const http = require('http')
 const cors = require('cors')
 
-const { VoyagerServer } = require('@aerogear/voyager-server')
-const { KeycloakSecurityService } = require('@aerogear/voyager-keycloak')
```

Comment:
this import is used on line 20

Reply:
marking this again as this is breaking. we need this import to create an auth instance